### PR TITLE
fix(collaborator-avatars): add keyboard accessibility

### PR DESCRIPTION
**Pull request by tjgupta**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @tjgupta has signed the CLA. Thanks for the pull request!

File: src/features/collaborator-avatars/CollaboratorAvatars.js [hunk 1]
```diff
@@ -55,20 +56,29 @@ class CollaboratorAvatars extends Component<Props> {
...
+        onClick();
+    };
+
     render() {
-        const { collaborators, maxDisplayedUserAvatars, containerAttributes, onClick } = this.props;
+        const { collaborators, maxDisplayedUserAvatars, containerAttributes } = this.props;
 
         return (
-            // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
             <a
```
Comment (priyajeet) [inline src/features/collaborator-avatars/CollaboratorAvatars.js]: Should this just be a button then? PlainButton?

File: src/features/collaborator-avatars/CollaboratorAvatars.js [hunk 2]
```diff
@@ -55,20 +56,29 @@ class CollaboratorAvatars extends Component<Props> {
             : `+${collaborators.length - maxDisplayedUserAvatars}`;
     }
 
+    handleActivate = (event: SyntheticKeyboardEvent<HTMLAnchorElement> | SyntheticMouseEvent<HTMLAnchorElement>) => {
+        const { onClick } = this.props;
+        // bail if it's a keyboard event but not the enter key
+        if (event.key && event.key !== KEYS.enter) {
```
Comment (priyajeet) [inline src/features/collaborator-avatars/CollaboratorAvatars.js]: redundant `event.key &&`

File: src/features/collaborator-avatars/CollaboratorAvatars.js (hunk 1 above)
Comment (cweeii) [reply src/features/collaborator-avatars/CollaboratorAvatars.js]: I agree.  Then you can also remove the role="button"

Comment (tjgupta) [reply src/features/collaborator-avatars/CollaboratorAvatars.js]: Yep, this is better. Done.

**Pull request merged**